Multisig/Features/Connect to Web/UI/List/WebConnectionsViewController.swift

```diff
@@ -212,6 +212,10 @@ extension WebConnectionsViewController: QRCodeScannerViewControllerDelegate {
 }
 
 extension WebConnectionsViewController: NavigationRouter {
+    func routeFrom(from url: URL) -> NavigationRoute? {
```

Comment:
Can you add a comment why this returns nil for now?

Reply:
it's nil because this router is not used to get the url -> navroute conversion.

The method is not used on this instance (and others).